src/bcrhp/bcrhp/migrations/operations/privileged_sql.py

```diff
@@ -0,0 +1,42 @@
+from django.db.migrations.operations.special import RunSQL
```

Comment:
Was this changed from the version in `/common/migrations/operations`? If not, please remove.

Reply:
My bad, having some logic only in common throws me off.